app/lib/link_checker/uri_checker/http_checker.rb

```diff
@@ -236,6 +242,18 @@ def make_request(method, check_ssl: true)
       end
 
       response
+    rescue ArgumentError => e
```

Comment:
We should have this as the last exception we rescue from since it's the most generic. This would mean that if any of the earlier exceptions inherit from ArgumentError they'd still be handled first. 

As it's pretty weird we should also put a comment above it to stay that it's an error type that Faraday doesn't handle.